controllers/releaseplan/controller.go

```diff
@@ -64,7 +64,6 @@ func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	return controller.ReconcileHandler([]controller.Operation{
 		adapter.EnsureMatchingInformationIsSet,
-		adapter.EnsureOwnerReferenceIsSet,
```

Comment:
I wanted to check if this should also be removed in the release controller and it's not there. Any idea why it's not there?